R/namespace.R

```diff
@@ -2,7 +2,7 @@
 namespace_imports <- function(path = find_package()) {
   imports <- tryCatch({
     pkg_name <- suppressWarnings(pkg_name(path))
-    data <- parseNamespaceFile(package = pkg_name, package.lib = file.path(path, ".."))
+    data <- parseNamespaceFile(package = "", package.lib = path)
```

Comment:
Internally this will call `file.path` with a zero-length string, using `path//NAMESPACE` as the path instead of `path/NAMESPACE`. Using `package = basename(path)` would be more robust.